internal/strace/strace.go

```diff
@@ -43,9 +43,12 @@ var (
 	// unlinkat(AT_FDCWD /app, 0x5569a7e83380 /app/vendor/composer/e06632ca, 0x200)
 	unlinkatPattern = regexp.MustCompile(`\S+ ([^,]+), 0x[a-f\d]+ ([^,]+), 0x[a-f\d]+`)
 
-	// This regex captures 1) the file path and 2) the write buffer and bytes written together. #2 will be parsed further before used.
-	// write(0x1 host:[5], 0x557f2cfde7c0 "django.template.base\nImporting django.template...", 0xe64)
-	writePattern = regexp.MustCompile(`\S+ ([^,]+), 0x[a-f\d]+ (".+)`)
+	// This regex parses just the file path. Bytes written is parsed further below as the nature of the write buffer makes it unideal to parse through regex.
+	// TODO: We can see how we can potentially reuse regex patterns.
+	writePattern = regexp.MustCompile(`\S+ ([^,]+),.*`)
+
+	// Length of the hex prefix "0x".
```

Comment:
Just make this `hexPrefix := "0x"` and call `len()` on it where you currently use `hexPrefixLength`. Then on line 181 you can use `hexPrefix` instead of `"0x"`

Reply:
Done.